components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -159,13 +214,14 @@ - (void)testSnackbarAccessibiltyViewIsModalShouldBeNoWithNoActions {
   XCTAssertFalse(self.manager.internalManager.overlayView.accessibilityViewIsModal);
```

Comment:
Same here, shouldn't this be an `XCTAssertTrue` given that it's supposed to be "YES"?

Reply:
Addressed in a5ddbbd01590490c2cebea9e302ad8fbdd774c28.